ui/cypress.json

```diff
@@ -0,0 +1,3 @@
+{
+  "baseUrl": "http://localhost:3000"
```

Comment:
Does Cypress support this being overridden at run time with a command line flag?

Reply:
yessir `CYPRESS_baseUrl=[your baseUrl] npm run cypress:open` or `npm run cypress:open --config "baseUrl=myUrl"`

Reply:
I'll just leave this here then:
https://github.com/nhsx/standards-registry/blob/master/.github/workflows/deploy-dev.yml#L165
https://github.com/nhsx/standards-registry/blob/master/.github/workflows/deploy-test.yml#L114